crates/revm/revm-inspectors/src/tracing/js/mod.rs

```diff
@@ -324,32 +327,31 @@ where
 
     fn step_end(
         &mut self,
-        interp: &mut Interpreter,
-        data: &mut EVMData<'_, DB>,
-        eval: InstructionResult,
+        _interp: &mut Interpreter<'_>,
+        _data: &mut EVMData<'_, DB>,
     ) -> InstructionResult {
         if self.step_fn.is_none() {
             return InstructionResult::Continue
         }
 
-        if matches!(eval, return_revert!()) {
-            let db = EvmDb::new(data.journaled_state.state.clone(), self.to_db_service.clone());
-
-            let step = StepLog {
-                stack: StackObj(interp.stack.clone()),
-                op: interp.current_opcode().into(),
-                memory: MemoryObj(interp.memory.clone()),
-                pc: interp.program_counter() as u64,
-                gas_remaining: interp.gas.remaining(),
-                cost: interp.gas.spend(),
-                depth: data.journaled_state.depth(),
-                refund: interp.gas.refunded() as u64,
-                error: Some(format!("{:?}", eval)),
-                contract: self.active_call().contract.clone(),
-            };
-
-            let _ = self.try_fault(step, db);
-        }
+        // if matches!(eval, return_revert!()) {
```

Comment:
Not immediately obvious how we're supposed to do this now that the result isn't passed. Any ideas @rkrasiuk? Might have missed something in the new API.

Reply:
Answered in https://github.com/paradigmxyz/reth/pull/5032#discussion_r1360770302; pending fix.